connectivity/netsocket/tests/TESTS/network/emac/emac_TestNetworkStack.h

```diff
@@ -84,7 +84,9 @@ class EmacTestNetworkStack : public OnboardNetworkStack, private mbed::NonCopyab
          *
          * @return              MAC address as "V:W:X:Y:Z"
          */
-        virtual char *get_mac_address(char *buf, nsapi_size_t buflen);
+        //virtual char *get_mac_address(char *buf, nsapi_size_t buflen);
```

Comment:
This function is still defined in the implementation file isn't it: https://github.com/ARMmbed/mbed-os/blob/40c038555dc191100231aa256181a302cc983e43/connectivity/netsocket/tests/TESTS/network/emac/emac_TestNetworkStack.cpp#L153 ?

Reply:
Absolutely, thanks for noticing this! Fixed in the following commit.